### PR TITLE
fix: [CI-3475]: disable trigger tab when branchName is not defaultBra…

### PR DESCRIPTION
**Pull request by mtran7**
…nchName

##### Screenshots:

image
image

image

Access by Direct URL
image

A follow up PR to handle remaining git sync trigger flow: https://harness.atlassian.net/browse/CI-3522

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by mtran7** [comment]
retrigger titlecheck

**Comment by mtran7** [comment]
SonarQube issue: 
image
Follow Up Ticket
https://harness.atlassian.net/browse/PIE-2813

**Comment by mtran7** [comment]
@kapilkumawat86 could you please confirm that this should say `default` and not `main` on this doc? `Triggers can be added to a Pipeline in the main branch only.
`

https://ngdocs.harness.io/article/utikdyxgfz-harness-git-experience-overview#limitations

**Review by vardanbansal-harness — DISMISSED** [review]

File: src/modules/70-pipeline/pages/pipeline-details/__tests__/__snapshots__/PipelineDetails.test.tsx.snap [hunk 1]
```diff
@@ -144,7 +144,7 @@ exports[`Pipeline Details tests render snapshot view for non pipeline studio rou
         inputSetsText
       </a>
       <a
-        class="TabNavigation--tags TabNavigation--small"
+        class="TabNavigation--tags TabNavigation--disabled TabNavigation--small"
```
Comment (mtran7) [inline src/modules/70-pipeline/pages/pipeline-details/__tests__/__snapshots__/PipelineDetails.test.tsx.snap:147]: Need to add another case when trigger tab is not disabled

**Review by kapilkumawat86 — DISMISSED** [review]

**Comment by kapilkumawat86** [comment]
> @kapilkumawat86 could you please confirm that this should say `default` and not `main` on this doc? `Triggers can be added to a Pipeline in the main branch only.
> `
> Also does not need to be bolded
> 
> https://ngdocs.harness.io/article/utikdyxgfz-harness-git-experience-overview#limitations
> 
> CC: @michaelcretzman 

Yes we say default branch

**Review by kapilkumawat86 — APPROVED** [review]

**Review by vardanbansal-harness — APPROVED** [review]

**Pull request merged**